dhall-toml/src/Dhall/TomlToDhall.hs

```diff
@@ -0,0 +1,187 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE OverloadedLists #-}
+
+module Dhall.TomlToDhall
```

Comment:
A module header would be nice to have. One question it could answer is: Can any valid `TOML` object be converted to a Dhall `Expr`?